app/views/clients/_form.html.slim

```diff
@@ -13,27 +13,27 @@
           = f.input :last_name
       .row
         .col-xs-12.col-md-6
-          = f.input :date_of_birth, as: :date, start_year: Date.today.year - 90,
-                                 end_year: Date.today.year,
-                                 order: [:day, :month, :year],
-                                 include_blank: true
+          = f.input :date_of_birth, as: :date,
+            start_year: Date.today.year, end_year: Date.today.year - 100,
+               order: [:day, :month, :year],
+            include_blank: true
 
         .col-xs-12.col-md-6
           = f.input :gender,
-                    :collection => gender_collection,
+                    collection: Client.gender_collection,
```

Comment:
As I said MVC. Normally people would use a enum for such, but @kitsane was against enums because they're only stored with numbers in the db